pkg/run/storage.go

```diff
@@ -37,8 +37,8 @@ type RunStorageService struct {
 
 const (
```

Comment:
SEAWEED_ ...

Reply:
These are actually no longer used, so will just delete them (the client still provides them but the server doesn't care.

Reply:
removed